src/test/java/com/tom/test/excercise02/SearchCategoryTest.java

```diff
@@ -0,0 +1,34 @@
+package com.tom.test.excercise02;
+
+import com.tom.driver.manager.DriverUtils;
+import com.tom.page.object.Excercise02Page;
+import com.tom.test.TestBase;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+import static com.tom.navigation.PageURLs.EXCERCISE_02_URL;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class SearchCategoryTest extends TestBase {
+
+    @Test
+    @DisplayName("Checking if searching and choosing category return expected results")
+    void checkingIfSearchCategoryFieldReturnsExpectedResults() {
+        //given
+        Excercise02Page page = new Excercise02Page();
+        page.log().info("Navigating to page {}.", EXCERCISE_02_URL);
+        DriverUtils.navigateToPage(EXCERCISE_02_URL);
+
+        //when
+        String categoryText = page.getCategoryText(1);
+        page.log().info("Typing category text first 3 letters in uppercase, submitting and getting results.");
```

Comment:
trochę dużo logów dodajesz niepotrzebnie. Zaciemniają tylko test. Kod ma być tak napisany że jak test się wywali i dostaniesz w której linii to będziesz mniej więcej wiedzał co nie działa